docs/team/ptejasv.md

```diff
@@ -0,0 +1,14 @@
+# Paramita Tejasvi - Project Portfolio Page
+## Overview
```

Comment:
Isn't this the overview of your contributions rather than that of the project?

Reply:
I think this refers to the project though

![Screenshot 2021-11-04 at 3 32 57 PM](https://user-images.githubusercontent.com/69339443/140274429-0d731563-39a0-4fde-a525-e56541923125.png)

Reply:
Right. Sorry for the confusion.